ztools/x509/json.go

```diff
@@ -221,7 +224,33 @@ func (c *Certificate) MarshalJSON() ([]byte, error) {
 	jc.Subject = c.Subject
 	jc.SubjectDN = c.Subject.String()
 	jc.SubjectKeyInfo.KeyAlgorithm = c.PublicKeyAlgorithm
-	jc.SubjectKeyInfo.SPKIFingerprint = c.SPKIFingerprint
+
+	// Include all subject names, DNS names there are
+	for _, obj := range c.Subject.Names {
+
+		switch name := obj.Value.(type) {
+		case string:
+
+			if name[0] == '*' {
+				name = name[2:]
+			}
+			// Check that this is actually a url and not something else
+			if govalidator.IsURL(name) {
+				jc.Names = append(jc.Names, name)
+			}
+		}
+	}
+
+	for _, name := range c.DNSNames {
+
+		if name[0] == '*' {
+			name = name[2:]
+		}
```

Comment:
Do just lose the wildcard here then? Shouldn't something like `*.zmap.io` be what's added?